.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,63 @@
+name: 🐞 New Bug Report
+description: Create a bug report
+title: "🐞[Bug]: "
+labels: [Bug]
+assignees: "jcaillon"
+body:
+  - type: markdown
+    attributes:
+      value: |
+       Thank you for taking the time to fill out this bug report!
+       Fields marked with an asterisk (*) are required, and remaining fields are optional, but we encourage their usage.
+  - type: textarea
+    id: feature
+    attributes:
+      label: Please enter a clear and concise description of the issue.
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: What Operating System (OS)?
+      placeholder: e.g. Windows 10, Windows 11, MacOS, Linux, Any
+    validations:
+      required: true
```

Comment:
It will always be windows :)

```suggestion
```

Reply:
>> LGTM but images under docs/ were modified for no reasons, could you revert those changes?

I don't know how that happen.  I had to download the original 3P, and run WinMerge to get all the missing files.

>> It will always be windows :)

I thought there was a Linux port to NPP, but you're right. It's only Windows.

I fixed all the above issues, but when i look at [**view reviewed changes**], I see a yellow dotted square next to the image files. I believe that indicate that the file is the same.